.tekton/basic-component-pipeline.yaml

```diff
@@ -257,6 +239,10 @@ spec:
     - name: ADDITIONAL_TAGS
       value:
       - konflux-$(params.revision)
+    - name: SOURCE_ARTIFACT
+      value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
+    - name: CACHI2_ARTIFACT
+      value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
```

Comment:
It does not look like these are needed for `apply-tags` task.
https://github.com/konflux-ci/build-definitions/blob/main/task/apply-tags/0.1/apply-tags.yaml

Reply:
Good catch, removed in https://github.com/stackrox/stackrox/pull/11616/commits/21469bc3ca7b0a706d1adbebf5ac316117177184